include/geode/model/representation/core/mapping.h

```diff
@@ -42,6 +42,10 @@ namespace geode
     public:
         using Mapping = MappingType;
 
+        ModelMapping() = default;
+        ModelMapping( ModelMapping&& other ) = default;
+        ModelMapping& operator=( ModelMapping&& other ) = default;
+
```

Comment:
Still needed? If so, what about the copy constructor? delete or default?

Reply:
Not needed but I think they clarify the use, and they might clarify the compilation issues on the next shady uses.
I think we can delete the copy constructors, since the member is not copy-constructible